### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by hhhhhhalf** [xl-tang3/RCOT#5]

**Comment by xl-tang3** [comment]
Yes, SwinIR can indeed serve as the Tnet, but the implementation of the two-pass mechanism necessitates the incorporation of corresponding residual encoding layers within SwinIR.

---- Replied Message ----
| From | ***@***.***> |
| Date | 12/06/2024 19:31 |
| To | xl-tang3/RCOT ***@***.***> |
| Cc | Subscribed ***@***.***> |
| Subject | [xl-tang3/RCOT] Some questions (Issue #5) |

Hello, after reading the paper, may I ask if this approach is applicable to the window attention based transformer model, combined as a new T_net?

**Comment by hhhhhhalf** [comment]
Okay, I understand now. Thank you for your reply!
